stripe/src/main/java/com/stripe/android/model/parsers/PaymentIntentJsonParser.kt

```diff
@@ -59,7 +59,6 @@ internal class PaymentIntentJsonParser : ModelJsonParser<PaymentIntent> {
 
         return PaymentIntent(
             id = id,
-            objectType = objectType,
```

Comment:
Did you intend on removing this? Looks like you're still pulling `objectType` into a val above. One more occurrence of this below too.

Reply:
Yes, this is a safeguard that we are only attempting to parse the JSON representations of PaymentIntent objects. It's not useful to have this property beyond parsing; the fact that the object is a `PaymentIntent` means that `objectType` must be `payment_intent`.